### PR TITLE
Week 7 - Adding a photo (Aditya)

### PR DESCRIPTION
**Pull request by thisisaditya17**
Add profile picture of Aditya

### CODE REVIEW

File: docs/images/adi_profile_picture.png
Comment (Darren-Tung) [inline docs/images/adi_profile_picture.png:1]: The file is named wrongly. It should [github_username_in_lower_case].png.

**Review by Darren-Tung — COMMENTED** [review]
You have to change the file name and let me know

File: docs/images/adi_profile_picture.png
Comment (thisisaditya17) [reply docs/images/adi_profile_picture.png:1]: I did it. Can you check again?

**Review by Darren-Tung — APPROVED** [review]
LGTM

**Pull request merged**